src/azure-cli/azure/cli/command_modules/profile/__init__.py

```diff
@@ -40,43 +40,67 @@ def load_command_table(self, args):
-            c.argument('identity_port', type=int, help="the port to retrieve tokens for login. Default: 50342", arg_group='Managed Service Identity')
+
+            # Device code flow
             c.argument('use_device_code', action='store_true',
-                       help="Use CLI's old authentication flow based on device code. CLI will also use this if it can't launch a browser in your behalf, e.g. in remote SSH or Cloud Shell")
-            c.argument('use_cert_sn_issuer', action='store_true', help='used with a service principal configured with Subject Name and Issuer Authentication in order to support automatic certificate rolls')
-            c.argument('scopes', options_list=['--scope'], nargs='+', help='Used in the /authorize request. It can cover only one static resource.')
-            c.argument('client_assertion', options_list=['--federated-token'], help='Federated token that can be used for OIDC token exchange.')
+                       help="Use device code flow. CLI will also use this if it can't launch a browser, "
+                            "e.g. in remote SSH or Cloud Shell.")
+
+            # Service principal
+            c.argument('service_principal', action='store_true',
+                       help='Log in with a service principal.')
+            c.argument('use_cert_sn_issuer', action='store_true',
+                       help='Use Subject Name + Issuer (SN+I) authentication in order to support automatic '
+                            'certificate rolls.')
+            c.argument('client_assertion', options_list=['--federated-token'],
+                       help='Federated token that can be used for OIDC token exchange.')
+
+            # Managed identity
+            c.argument('identity', options_list=('-i', '--identity'), action='store_true',
+                       help="Log in using managed identity", arg_group='Managed Identity')
 
         with self.argument_context('logout') as c:
             c.argument('username', help='account user, if missing, logout the current active account')
             c.ignore('_subscription')  # hide the global subscription parameter
 
         with self.argument_context('account') as c:
-            c.argument('subscription', options_list=['--subscription', '-s', '--name', '-n'], arg_group='', help='Name or ID of subscription.', completer=get_subscription_id_list)
+            c.argument('subscription', options_list=['--subscription', '-s', '--name', '-n'],
+                       completer=get_subscription_id_list, help='Name or ID of subscription.')
             c.ignore('_subscription')
 
         with self.argument_context('account list') as c:
-            c.argument('all', help="List all subscriptions from all clouds, rather than just 'Enabled' ones", action='store_true')
+            c.argument('all', action='store_true',
+                       help="List all subscriptions from all clouds, rather than just 'Enabled' ones", )
```

Comment:
```suggestion
                       help="List all subscriptions from all clouds, including 'disabled' clouds", )
```

Reply:
@jiasli is 'from all clouds' accurate?

```suggestion
                       help="List all subscriptions from the current environment, rather than just 'Enabled' ones", )
```

Reply:
> is 'from all clouds' accurate?

Yes: https://github.com/Azure/azure-cli/blob/41c2910979d651505bc6f2657eebd5ff7e58e79a/src/azure-cli/azure/cli/command_modules/profile/custom.py#L54

I will change it to

> List all subscriptions from all clouds, including subscriptions that are not 'Enabled'.